core/common/src/serializers/LocalDateTimeSerializers.kt

```diff
@@ -33,7 +33,7 @@ public object LocalDateTimeIso8601Serializer: KSerializer<LocalDateTime> {
 }
 
 /**
- * A serializer for [LocalDateTime] that represents the values as their components.
+ * A serializer for [LocalDateTime] that represents the value as its components.
```

Comment:
> the value

I believe it's incorrect as there's no implied value here. In "the values", `the` represents the implied `LocalDateTime`, so is equivalent to "the LocalDateTime values". It would be better to either keep it as it were, or use "a value".